Tools/flatpak/flatpakutils.py

```diff
@@ -817,7 +817,12 @@ def run_in_sandbox(self, *args, **kwargs):
 
             if args[0] == "bash":
                 args.extend(['--noprofile', '--norc', '-i'])
-                sandbox_environment["PS1"] = f"[📦🌐🐱 $FLATPAK_ID {self.platform}@{self.build_type} \\W]\\$ "
+                sandbox_environment["PS1"] = f"[$FLATPAK_ID {self.platform}@{self.build_type} \\W]\\$ "
+                try:
+                    if 'en_US.utf8' in subprocess.check_output(['locale', '-a']).decode("utf8").split():
```

Comment:
Any utf8 locale could be used, so no need to gate this on the en_US one.

Reply:
No, I have gl_ES.utf8 as default and it does not work if I don't have en_US.utf8 available